src/pages/home_page/HomePage.jsx

```diff
@@ -19,10 +19,11 @@ export const HomePage = () => {
       user_pic:
         "https://img.freepik.com/free-vector/gamer-mascot-geek-boy-esports-logo-avatar-with-headphones-glasses-cartoon-character_8169-228.jpg",
       user_handle: "Marvel",
-      user_name: "Tanay Pratap",
+      user_name: "Tony Stark",
       points: 280,
     },
   ];
+  if (!quizzes) return <Loader />;
```

Comment:
I think it would be better syntax to have 1 return statement and have condition inside it. Rather than 2 returns on Line 26 and 27

Reply:
okay i will look into it